sdk/storage/azure-storage-files-datalake/inc/azure/storage/files/datalake/datalake_options.hpp

```diff
@@ -731,45 +726,6 @@ namespace Azure { namespace Storage { namespace Files { namespace DataLake {
     Azure::Core::Nullable<std::string> ExpiresOn;
   };
 
-  /**
-   * @brief Optional parameters for FileClient::DownloadToBuffer and FileClient::DownloadToFile.
-   */
-  struct DownloadFileToBufferOptions
```

Comment:
why was this deleted?

Reply:
We don't have any API that's using these options anymore.